python/paddle_fl/mpc/layers/tensor.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+basic tensor layers.
+"""
+import six
+import numpy
+from paddle.fluid.data_feeder import check_type, check_dtype
+from paddle.fluid.param_attr import ParamAttr
+from paddle.fluid.initializer import Initializer
+
+from ..framework import MpcVariable
+from ..framework import check_mpc_variable_and_dtype
+from ..mpc_layer_helper import MpcLayerHelper
+
+
+__all__ = [
+    'create_mpc_parameter',
+]
+
+
+def create_mpc_parameter(shape,
```

Comment:
why not directly call helper.create_mpc_parameter?

Reply:
follow PaddlePaddle's style. add function 'create_mpc_parameter' in layers.